FrontEnd/src/components/loader/Loader.jsx

```diff
@@ -0,0 +1,36 @@
+import css from "./Loader.module.css";
```

Comment:
Could we add a loader to the page where we fetch regions?
Also, I would really appreciate it if you add some screenshots to the pr description

Reply:
Screenshot added to the description of the PR :)